docs/Dialects/FIRRTL/FIRRTLIntrinsics.md

```diff
@@ -94,3 +94,108 @@ The enable input is sampled at the rising edge of the input clock; any changes o
 | in   | input     | Clock    | input clock                 |
 | en   | input     | UInt<1>  | enable for the output clock |
 | out  | output    | Clock    | gated output clock          |
+
+### circt.chisel_assert_assume
+
+Generate a clocked SV assertion with companion assume statement.
+
+Has legacy special behavior and should not be used by new code.
+
+| Parameter | Type   | Description                                                                         |
+| --------- | ------ | ----------------------------------------------------------------------------------- |
+| format    | string | Format string per SV 20.10, 21.2.1.  Optional.                                      |
+| label     | string | Label for assert/assume.  Optional.                                                 |
+| guards    | string | Semicolon-delimited list of pre-processor tokens to use as ifdef guards.  Optional. |
```

Comment:
Would be nice to mention guards are only applied to the companion assume

Reply:
They're actually not, the companion assume magically gets an extra hardcoded guard.  Hmm.

Reply:
(Just double-checked :sweat_smile: :+1: )